src/main/plugin-metadata/plugin-security.policy

```diff
@@ -12,5 +12,4 @@ grant codeBase "${codebase.zstd-jni}" {
 
 grant codeBase "${codebase.qat-java}" {
   permission java.lang.RuntimePermission "loadLibrary.*";
-  permission org.opensearch.secure_sm.ThreadPermission "modifyArbitraryThread";
```

Comment:
@mulugetam I missed that in the original pull request, why you added this permission? It should not be granted.